sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -60,6 +58,18 @@
 # regions.
 PRICING_TABLE_URL_FMT = 'https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/{region}/index.csv'  # pylint: disable=line-too-long
 
+regions_enabled: Set[str] = None
+
+
+def get_enabled_regions() -> Set[str]:
+    global regions_enabled
+    if regions_enabled is None:
+        aws_client = aws.client('ec2', region_name='us-east-1')
+        regions_enabled = aws_client.describe_regions()['Regions']
+        regions_enabled = {r['RegionName'] for r in regions_enabled}
+        regions_enabled = regions_enabled.intersection(set(ALL_REGIONS))
```

Comment:
Does the github runner's credentials enable all regions in ALL_REGIONS?

Reply:
Yes, it is an IAM role in my account that enables all the regions.